kats/src/main/kotlin/kats/Option.kt

```diff
@@ -24,6 +24,11 @@ package kats
  */
 sealed class Option<out A> {
 
+    companion object {
+
+        inline operator fun <A> invoke(f: () -> A): Option<A> = f().let { if (it == null) None else Some(it) }
```

Comment:
And what is the way if we want to compute with Option like the result? We should manage this every time?